docs/resources/rds_instance.md

```diff
@@ -281,6 +284,15 @@ The `volume` block supports:
   Changing this parameter will create a new resource. For details about volume types, see
   [DB Instance Storage Types](https://support.huaweicloud.com/intl/en-us/productdesc-rds/rds_01_0020.html).
 
+The `restore` block supports:
+
+* `instance_id` - (Required, String, ForceNew) Specifies the DB instance ID.
+
+* `backup_id` - (Required, String, ForceNew) Specifies the ID of the backup used to restore data.
+
+* `database_name` - (Optional, Map, ForceNew) Specifies the database to be restored. This parameter applies only to
+  Microsoft SQL Server databases.
+
```

Comment:
put this at another place, this looks like `disk_encryption_id` is  in the restore block

Reply:
done